### DISCUSSION TITLE
What's about private members of classes?

### DISCUSSION THREAD

**Discussion by fussel178** [fliegwerk/fliegdoc#5] [Ideas] [answered]
First of all,
really really nice tool! We already use it in our project (which is a monorepo) and it is working beautifully!
As I looked around, I noticed that all members of a class are exported regardless if the member is public or private.
As far as I now, fliegdoc should only generate documentation for exported types.
Is a private member of a class an exported type?

For example, we have the following class:
```ts
export class MyClass {
  /**
   * This is my private value!
   */
  private myPrivateValue: string;

  /**
   * this is my public function hello!
   */
  myFunction(): void {
    console.log('Hello World');
  }
}
```

Which generates to:
![Screenshot_20210107_182507](https://user-images.githubusercontent.com/52416718/103924119-87d15600-510d-11eb-873a-faa228f1b34d.png)

Any ideas? :)

**Comment by pklaschka** [comment]
This is now implemented and will get released in the next release 😃

**Comment by pklaschka** [comment, marked as answer]
Now released in v0.4.0: https://github.com/fliegwerk/fliegdoc/milestone/3?closed=1
